### PR TITLE
Phone UI 📱

### PR DESCRIPTION
**Pull request by revett**
### Problem

There is currently no UI for the phone, only a sidebar.

### Solution

- Add phone UI, with both presentational and container **Phone** components.
- Move the logic to hold the state for which video is selected to the **App** container component, as this state is required by both the Sidebar and Phone parts of the page.
- This meant that the **Sidebar** container component is no longer required.

image

image

image

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

**Comment by revett** [comment]
[[SSA 0.8.0] Phone UI](https://trello.com/c/z6TqCh6w/1163-ssa-0-8-0-phone-ui)

File: src/components/container/app.js [hunk 1]
```diff
@@ -1,32 +1,80 @@
...
       this.setState({ error: true });
     });
   }
+  onSelectedVideoChange(shortHash) {
+    this.setState({ selectedVideoShortHash: shortHash });
+  }
+  getPageContent() {
+    if (this.state.error) {
+      return (
+        <h1>ERROR</h1> // TODO!
```
Comment (revett) [inline src/components/container/app.js]: Handled in a different PR.

File: src/components/container/app.js [hunk 2]
```diff
@@ -1,32 +1,80 @@
...
+      return (
+        <PhoneContainer
+          brand={this.state.brand}
+          selectedVideo={this.getSelectedVideo()}
+        />
+      );
+    }
+
+    return (
+      <h1>LOADING</h1> // TODO!
```
Comment (revett) [inline src/components/container/app.js]: Handled in a different PR.

File: src/components/presentational/phone.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    this.state = {};
+    this.requestFullscreen = this.requestFullscreen.bind(this);
+  }
+  componentDidMount() {
+    EventMachine.on(
+      'player:fullscreen',
+      this.requestFullscreen,
+    );
+  }
+  requestFullscreen() {
```
Comment (revett) [inline src/components/presentational/phone.js]: This is used by the **Fullscreen** button which will be added in a future PR.

File: src/components/presentational/phone.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    const classes = classnames(
+      [
+        'outline',
+        this.props.videoPlatform,
+        this.props.selectedVideo.orientation,
+      ],
+    );
+
+    return (
+      <div className="Phone">
```
Comment (revett) [inline src/components/presentational/phone.js]: I thought about breaking this down into lots of little components, however I think getting the transition complete first is more important.

File: src/components/presentational/phone.js (hunk 3 above)
Comment (stevenjack) [inline src/components/presentational/phone.js]: I thought we'd removed full screen?

File: src/components/presentational/phone.js (hunk 4 above)
Comment (stevenjack) [inline src/components/presentational/phone.js]: Yep agree, PR is big enough 👍

**Pull request merged**